app/signals/apps/signals/models/category.py

```diff
@@ -188,10 +188,6 @@ def has_icon(self):
     def clean(self):
         super().clean()
 
-        if self.pk and self.slug:
```

Comment:
In regards to what I said in my other comment, I think it will be good to keep this ValidationError in combination with the feature flag